aexpect/ops.py

```diff
@@ -0,0 +1,616 @@
+# This Python file uses the following encoding: utf-8
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Intra2net AG and aexpect contributors
+
+"""
+
+SYNOPSIS
```

Comment:
Docstring is still not following pep257 "multi line" section.

Reply:
Sorry, I didn't resolve the older comment yet. Will resolve it now by:
1) dropping all headers here
2) turning the single summary line under the SYNOPSIS header into the one-liner defined by pep257
3) drop the DESCRIPTION for the multi-line defails defined by pep257